colibre/auto_plotter/stellar_mass_halo_mass.yml

```diff
@@ -588,3 +588,152 @@ stellar_mass_halo_mass_M200_centrals_30:
     - filename: GalaxyStellarMassHaloMass/Moster2013.hdf5
     - filename: GalaxyStellarMassHaloMass/Schaye2015_Ref_100_M200.hdf5
     - filename: GalaxyStellarMassHaloMass/Read2017.hdf5
+
+stellar_mass_halo_mass_M200_centrals_projected_10:
+  type: "scatter"
+  select_structure_type: 10
+  comment: "Centrals only"
+  legend_loc: "upper left"
+  x:
+    quantity: "masses.mass_200crit"
+    units: Solar_Mass
+    start: 1e13
+    end: 5e15
+  y:
+    quantity: "derived_quantities.projected_stellar_mass_10_kpc"
```

Comment:
Does the field `derived_quantities.projected_stellar_mass_10_kpc` actually exist? I don't see it defined in the registration file.

The same comment applies to `derived_quantities.projected_stellar_mass` measured in apertures of different size.

Reply:
I had forgotten to actually add this (critical part), thanks for noticing! These derived projected stellar masses, which I have now added, are just averages over the three axes.